src/domain/schema.graphql

```diff
@@ -188,3 +198,13 @@ type ContractInfo @entity {
 	version: String!
 	address: Bytes!
 }
+
+type Event @entity {
+	id: ID!
+	type: EventType!
+	data: String!
```

Comment:
what is this data field ? why it is needed ?

Reply:
That include some JSON data Alchemy might need. @jellegerbrandy

Reply:
@jellegerbrandy ?

Reply:
is that cannot be analyzed from the event it self ?

Reply:
@jellegerbrandy